package.json

```diff
@@ -89,7 +91,7 @@
     "oldie": "^1.3.0",
     "postcss-pseudo-classes": "^0.2.0",
     "postcss-scss": "^2.0.0",
-    "puppeteer": "^1.19.0",
+    "puppeteer": "19.3.0",
```

Comment:
can't quite believe how far behind we were, guess maybe puppeteer changed their version numbering